### ISSUE TITLE
How do you stop white T-Shirts becoming discolored?

### ISSUE THREAD

**Issue by Morningstar88** [Microflow/Eagle-Shirts#8]
I saw a lot of white TShirts on the MRT recently. They were more popular with women?

So how do you stop white TShirts turning yellow?

We need to include some sort of care card/list of tips for customers.

**Comment by Morningstar88** [comment]
## Google Search

https://www.google.com/search?q=how+to+stop+white+tshirts+going+yellow&oq=how+to+stop+white+tshirts+going+yellow&aqs=chrome..69i57j0l5.7119j0j7&sourceid=chrome&ie=UTF-8

## General Info

What causes white shirts to turn yellow?

Even white fabrics made from natural fibers like cotton and linen can turn yellow if they are exposed to too much chlorine bleach. Overuse of chlorine bleach can also cause white clothes to yellow when hung in the sun to dry because the sun adds another layer of bleaching due to ultra-violet rays.

**Comment by taiwanluthiers** [comment]
Unfortunately there isn't a lot we can do on our end. It's up to the end user to take care of it... Meaning we can guarantee it's white coming out but if the customer wash it a million times, and it turns yellow, then it's not our problem.

**Comment by Morningstar88** [comment]
Did you read the Google search I linked? There are tons of things we/people
can do. The first is to include a care card, like I said in the OP.

It *is* our problem, because an unhappy customer won't buy from us again.

It would help if you were more constructive and proactive with your
feedback.

**Comment by Morningstar88** [comment]
Some good ideas here:

https://www.mamaslaundrytalk.com/have-your-white-clothes-turned-yellow/

**Comment by taiwanluthiers** [comment]
What I mean is, best we can do is maybe dye the clothing slightly blue to counteract the yellow, but every white T shirt I buy is white from the start but as I wear it and wash it, it turns yellow. At least the instructions you have given is things the end user does to make it not turn yellow...

The 7-11 next to the aroma bar has a T shirt printing printer, they just use a heat press and do not wash the shirt. You can't iron T shirts because it will mess up the ink. Before you ask, that printer cost 1 million NT...

**Comment by Morningstar88** [comment]
I agree it's mainly the end users responsibility, but if we include some
info that will help them.

There may also be some TShirts fabrics that are more resistant to yellow
staining.

**Comment by taiwanluthiers** [comment]
One solution is we select colors that look white but isn't white. For example light grey, very light yellow, etc. that fits with the design. That way they don't look bad from stains. 

Polyester I believe is more resistant to stains, but unless it's special cooling fabric (such as the Airism shirt that Uniqlo sells) it won't be comfortable to wear.

From now on I will buy my T shirt stock from here:

https://item.taobao.com/item.htm?spm=a1z09.2.0.0.4ea82e8dl70ArI&id=18995457711&_u=e2023lrtq570e0

They have lots of color to choose from, and is very good quality T shirt stock. Price is roughly 60-70nt a shirt.

**Comment by Morningstar88** [comment]
OK that's better. I was thinking pale green. And what about actual yellow?
lol. That colour is popular with women, and it solves a lot of problems.

We can research other fabrics another time.

Are those TShirts Chinese or Taiwanese made? If you can source a burgundy
one that would help long term

Next buy go with white, and yellow IMO.

**Comment by Morningstar88** [comment]
Their dark orange looks good.

**Comment by taiwanluthiers** [comment]
It's Taobao, obviously from China but the shirt itself says it was made in Vietnam...

**Comment by Morningstar88** [comment]
I was just checking. If you say "obviously" to someone it sounds like
you're talking down to them.

Im concerned about conditions in factories in Vietnam and China. Can we
source TShirts locally, even if we pay more?

**Comment by taiwanluthiers** [comment]
I wouldn't know. I see lots of T shirts sold in Taiwan at inflated prices and they claim it's made in Taiwan when they just bought it from Taobao. Some are shameless enough to rip the advertisements from Taobao where everything is written in simplified Chinese thinking Taiwanese are too dumb to notice. 

If we have to source shirts from Taiwan factories then we will pay a lot more, as much as 300nt per blank. I think that SunFlower brand is made in Taiwan.

**Comment by Morningstar88** [comment]
You made a good point about locals claiming TShirts are made in Taiwan. We
can go with China for now, and later try and source some local blanks. If
our shirts are good enough, people will pay more.

**Comment by Morningstar88** [comment]
## Getting T-Shirts made in Taiwan

Not directly relevant but worth saving for later.

https://tw.forumosa.com/t/getting-custom-t-shirts-tees-made-in-taipei/3832
